boot/seal_test.go

```diff
@@ -1009,3 +1009,32 @@ func (s *sealSuite) TestResealKeyToModeenvWithFdeHookSad(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(resealKeyToModeenvUsingFDESetupHookCalled, Equals, 1)
 }
+
+func (s *sealSuite) TestResealKeyToModeenvWithFdeHookVerySad(c *C) {
```

Comment:
I think Ian asked also for a test where HasFDESetupHook fails? (of course is not called anymore, but that's how the old code was failing)

Reply:
yes indeed I would like to make sure that doesn't accidentally happen again :-)

Reply:
Oh, sorry, I misunderstood. I added something now, I hope I got it right.